File: PDF.co Web API/Manage Background Jobs/Python/Check Job Status/ManageBackgroundJob.py
import requests

url = "https://api.pdf.co/v1/job/check"

payload = {'jobid': '123'}
files = [

]
headers = {
		'x-api-key': '{{x-api-key}}'
}

response = requests.request("POST", url, headers=headers, data = payload, files = files)

print(response.text.encode('utf8'))

File: Cloud API Server/Manage Background Jobs/JavaScript/ManageBackgroundJob.js
//*******************************************************************************************//
//                                                                                           //
// Get Your API Key: https://app.pdf.co/signup                                               //
// API Documentation: https://developer.pdf.co/api/index.html                                //
//                                                                                           //
// Note: Replace placeholder values in the code with your API Key                            //
// and file paths (if applicable)                                                            //
//                                                                                           //
//*******************************************************************************************//


// Please NOTE: In this sample we're assuming Cloud Api Server is hosted at "https://localhost". 
// If it's not then please replace this with with your hosting url.

var request = require('request');
var options = {
  'method': 'POST',
  'url': 'https://localhost/job/check',
  'headers': {
    'x-api-key': '{{x-api-key}}'
  },
  formData: {
    'jobid': '123'
  }
};
request(options, function (error, response) {
  if (error) throw new Error(error);
  console.log(response.body);
});
